### PR TITLE
docs: general improvements

### PR DESCRIPTION
**Pull request by anabastos**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -4,24 +4,49 @@ title: Installation
 sidebar_label: Installation
 ---
 
-Add via yarn
+### Pre-requisites
+
+#### BS-Platform
+ In order to start using Reform you need the basic tooling for ReasonML using [Buckescript](https://bucklescript.github.io/en/).
```
Comment (fakenickels) [inline docs/installation.md]:
```suggestion
 In order to start using Reform you need the basic tooling for ReasonML using [BuckleScript](https://bucklescript.github.io/en/).
```

**Comment by anabastos** [comment]
@fakenickels Fixed

File: docs/installation.md [hunk 2]
```diff
@@ -4,24 +4,49 @@ title: Installation
...
+
+#### React
+Reform is a library for React applications, so we assume that you are already familiar with [React](https://reactjs.org/).
+
+### Instalation and Setup
+
+You’ll need to have [Node 8.16.0 or Node 10.16.0 or later version](https://nodejs.org/en/) on your local development machine (but it’s not required on the server). You can use `nvm` (macOS/Linux) or `nvm-windows` to easily switch Node versions between different projects.
+
+If you want to create a ReasonReact development envinroment you can use the create-react-app to bootstrap a initial project.
+`yarn create react-app my-app --scripts-version reason-scripts`
```
Comment (fakenickels) [inline docs/installation.md]: just missing to use `bsb` templates here instead!
```suggestion
`bsb -init lambda5_onegraph -theme react-hooks`
```

**Comment by fakenickels** [comment]
All very good! Just missing the part about using `bsb -init` and we're good to merge 🎉

**Comment by fakenickels** [comment]
@all-contributors please add @anabastos for docs

**Pull request merged**